codegen/src/test/resources/software/amazon/awssdk/codegen/poet/eventstream/eventone_.java

```diff
@@ -0,0 +1,55 @@
+package software.amazon.awssdk.services.jsonprotocoltests.model.eventstream;
+
+import software.amazon.awssdk.annotations.Generated;
+import software.amazon.awssdk.annotations.SdkInternalApi;
+import software.amazon.awssdk.services.jsonprotocoltests.model.EventOne;
+import software.amazon.awssdk.services.jsonprotocoltests.model.EventStreamOperationResponseHandler;
+
+/**
+ * A specialization of {@code software.amazon.awssdk.services.jsonprotocoltests.model.EventOne} that represents the
+ * {@code EventStream#EventOne} event. Do not use this class directly. Instead, use the static builder methods on
+ * {@link software.amazon.awssdk.services.jsonprotocoltests.model.EventStream}.
+ */
+@SdkInternalApi
+@Generated("software.amazon.awssdk:codegen")
+public final class EventOne_ extends EventOne {
```

Comment:
Hmm, not a big fan of the underscore in the end. Can we use something else? Or rename the base class to `BaseEventOne`?

Reply:
Hmm have a suggestion for an alternative suffix/prefix?

Reply:
Settled on "Default" as a prefix